Source/WebCore/page/Page.h

```diff
@@ -1181,6 +1181,7 @@ class Page : public RefCountedAndCanMakeWeakPtr<Page>, public Supplementable<Pag
 
     void opportunisticallyRunIdleCallbacks(MonotonicTime deadline);
     void performOpportunisticallyScheduledTasks(MonotonicTime deadline);
+    void deleteRemovedChildNodes();
```

Comment:
I don't think "child" adds any value here. Just call it deleteRemovedNodes or maybe executeDelayedNodeDeletionNow is more clear.